cmd/juju/commands/bootstrap.go

```diff
@@ -419,18 +421,27 @@ var getBootstrapFuncs = func() BootstrapInterface {
 	return &bootstrapFuncs{}
 }
 
-var supportedJujuSeries = func() set.Strings {
+var supportedJujuSeries = func(now time.Time, bootstrapSeries string, dailyImageStream bool) (set.Strings, error) {
```

Comment:
If we extract this to core/series we won't need the duplicate function

Reply:
Exactly, I've done this in future commits.

Extracted to be more testable.